src/StarcounterClientFiles/bower-list.txt

```diff
@@ -1,18 +1,18 @@
 bower check-new     Checking for new versions of the project dependencies...
 starcounter-clientfiles#2.1.1 C:\Work\Starcounter\StarcounterClientFiles
-├─┬ bootswatch#3.3.7 (latest is 4.0.0-beta.3)
-│ └── bootstrap not installed
+├── Underwear.css#0.0.1
 ├─┬ imported-template#3.1.1
 │ └── juicy-html#3.0.0
 ├── juicy-redirect#0.6.0
-├─┬ palindrom-client#4.0.3
-│ ├── Palindrom#3.1.0
-│ └─┬ polymer#2.3.1 incompatible with ^1.11.0 (1.11.2 available, latest is 2.4.0-rc.1)
-│   ├── shadycss#1.1.0
-│   └── webcomponentsjs#1.0.22
+├─┬ palindrom-client#4.0.3 (latest is 5.0.0)
+│ ├── Palindrom#3.1.0 (latest is 5.0.0)
+│ └─┬ polymer#2.3.1 incompatible with ^1.11.0 (1.11.2 available, latest is 2.5.0)
+│   ├── shadycss#1.1.0 (1.1.1 available)
+│   └── webcomponentsjs#1.0.22 (1.1.0 available)
 ├── palindrom-redirect#1.0.0
-├── polymer#2.3.1 (2.4.0-rc.1 available)
+├── polymer#2.3.1 (2.5.0 available)
 ├─┬ starcounter-include#4.2.0
 │ ├── imported-template#3.1.1
 │ └── translate-shadowdom#0.0.5
-└── webcomponentsjs#1.0.22
+├── uniform.css#0.0.1
+└── webcomponentsjs#1.0.22 (1.1.0 available)
```

Comment:
Can you please also bump `palindrom-client` and `Palindrom` to 5.0.0, so SCF `3.x` gets equal with `2.x` in that regard?

`webcomponentsjs` and `Polymer` can be bumped another time.

Reply:
Sure